apps/ledger-live-desktop/src/renderer/families/bitcoin/AccountHeaderManageActions.ts

```diff
@@ -44,16 +49,20 @@ const AccountHeaderActions = ({ account, parentAccount }: Props) => {
   };
 
   return [
-    {
-      key: "Stake",
-      icon: IconCoins,
-      label: t("accounts.contextMenu.yield"),
-      event: "button_clicked2",
-      eventProperties: {
-        button: "stake",
-      },
-      onClick: stakeOnClick,
-    },
+    ...(availableOnStake
```

Comment:
I guess technically we shouldn't need to check the flag here, too, if we check in the main screens/account file below?

Reply:
i know i agree however there are lots of nested screens that display the stake/yield button, adding the check here too is to make sure no place has been missed on the codebase. (like this one that was missed for all the currencies!) , and considering the release timing i thought it would be good to cover all the possibilities (also screens that i don't know about) . Let me know if it doesn't sound fair though!